test/e2e/alm_e2e_test.go

```diff
@@ -132,21 +132,22 @@ func TestCreateInstallPlanManualApproval(t *testing.T) {
 		if step.Resource.Kind == "CustomResourceDefinition" {
 			_, err := c.GetCustomResourceDefinition(step.Resource.Name)
 
-			require.NoError(t, err)
+			require.Error(t, err)
 			vaultResourcesPresent = vaultResourcesPresent + 1
```

Comment:
@ecordell This fails because our CI target cluster always has all the CRDs. Should we just disable this check?

Reply:
It looks like it should be finding the already installed CRDs and counting them, right? I wouldn't expect this to fail